web/src/components/base/Input.tsx

```diff
@@ -55,14 +17,16 @@ export type InputProps = {
   value: string
   submitted?: boolean
   onChange: (input: string) => void
-  onKeyDown?: (e: React.KeyboardEvent<HTMLTextAreaElement | HTMLInputElement>) => void
+  onKeyDown?: (e: React.KeyboardEvent<HTMLDivElement>) => void
   multiline?: boolean
   numberOfLines?: number
   maxLength?: number
   required?: boolean
   hint?: string
-  hintIsLabel?: boolean
-  placeholder?: string
+  size?: 'small' | 'medium'
```

Comment:
❓ Why do we need the size? Shouldn't that already be handled by the `numberOfLines`?

Reply:
Speaking of which, shouldn't `numberOfLines` already cover the `multiline` prop? (I know that's not your change, but seeing it now, it's a little silly)

Reply:
> ❓ Why do we need the size? Shouldn't that already be handled by the `numberOfLines`?

No this size is the height of the text-field:
https://mui.com/material-ui/react-text-field/#sizes

🤔 I remember it was 'medium' now its normal 🫣

Reply:
> Ah, I see, thank you for the explanation! I don't think that's in our style guide, though, all fields there seem to be of the same height.

what about malte?